bot/exts/evergreen/wolfram.py

```diff
@@ -133,22 +133,22 @@ async def get_pod_pages(ctx: Context, bot: Bot, query: str) -> Optional[List[Tup
                     f"processing a wolfram request: {url_str}, Response: {json}"
                 )
                 await send_embed(ctx, message)
-                return
+                return None
 
             message = "Something went wrong internally with your request, please notify staff!"
             log.warning(f"Something went wrong getting a response from wolfram: {url_str}, Response: {json}")
             await send_embed(ctx, message)
-            return
+            return None
 
         if not result["success"]:
             message = f"I couldn't find anything for {query}."
             await send_embed(ctx, message)
-            return
+            return None
 
         if not result["numpods"]:
             message = "Could not find any results."
             await send_embed(ctx, message)
-            return
+            return None
```

Comment:
Do you prefer these over just `return`? I'm as impartial as can be, but I'd be interested in hearing why you prefer these.

Reply:
I am following the holy PEP 8
> Be consistent in return statements. Either all return statements in a function should return an expression, or none of them should. If any return statement returns an expression, any return statements where no value is returned should explicitly state this as `return None`, and an explicit return statement should be present at the end of the function (if reachable)

Reply:
Mmm I wasn't aware of this, thanks for explaining.